main.js

```diff
@@ -384,6 +384,11 @@ exports.main = function (options, callbacks) {
       mainWindow.setAttribute('toggledon', 'false');
       win.activeInstall = true;
     }
+
+    if (options.loadReason !== 'install') {
+      win.tabCenterStartup = true;
```

Comment:
Why not pass this as a parameter to `initWindow`, which would pass it to `addVerticalTabs` which would pass it to the `VerticalTabs` constructor, which would pass it to its `init` method?  (I'm always a little leery of setting attributes, because it's much easier to forget to reset them…)